hack/multi-node/bootkube-recover

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+GLOG_v=${GLOG_v:-1}
+HOST=${HOST:-c1}
+SELF_HOST_ETCD=${SELF_HOST_ETCD:-false}
+
+if [ ${SELF_HOST_ETCD} = "true" ]; then
```

Comment:
for clarity - not saying we should remove - just advocating for clearer script name